R/harmonicHMC.R

```diff
@@ -59,11 +61,11 @@ harmonicHMC <- function(n,
   if (sum(F %*% init + g > 0) < length(g)) {
     stop("Initial value x does not satisfy Fx + g >=0")
   }
-  samples <- matrix(nrow = n + burnin, ncol = ncol(F))
+  samples <- matrix(nrow = nSample, ncol = ncol(F))
   randomBounceTime <-
     ifelse(length(time) == 2, TRUE, FALSE)
   bounceDistances <- vector(mode = "list",
-                            length = ifelse(diagnosticMode, n + burnin, 0))
+                            length = ifelse(diagnosticMode, nSample + burnin, 0))
```

Comment:
In this case, is it odd to return bounce distances for all samples, including burn in, but not return the burn in samples.

Reply:
Good point, thanks. Fixed.